Yoshi/Yoshi/YoshiActionResult.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  YoshiActionResult.swift
+//  Pods
+//
+//  Created by Christopher Jones on 2/8/16.
+//
```

Comment:
I meant updating line #3 to say Yoshi and adding the copyright information on line #6.